_posts/2016-01-13-Get-Inspired-on-GitHub.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Get Inspired on GitHub
+excerpt: Learn how to search for API design patterns on GitHub.
+layout: post
+date: 2016-01-13 00:00:00 +0100
+author: zdenek
+published: true
+comments: true
+---
+
+Last June API Blueprint became recognized as a language on GitHub.
+This brought the syntax highlighting and repository statistics for API
+Blueprint. More importantly, it has also enabled the search for API design
+patterns.
+
+Let’s take a quick look on how we can search and perhaps learn from the API
+designs done by others.
+
+# Search
+
+To search for API Blueprint on GitHub simply go to
+
+[https://github.com/search](https://github.com/search)
+
+<img width="640" src="/images/2016-01-13-Get-Inspired-on-GitHub/search.png" alt="search" />
+
+And enter the `language:"API Blueprint"` as the query. Hitting the search will
+list all the GitHub repositories that contains at least one API Blueprint
```

Comment:
ah.  I feel bad. I have fixed it in another draft and then pasted the wrong one here.